File: qr-auth-frontend/src/app/_model/response/qr-code-verification-response.ts
import {VerificationStatusEnum} from '../enums/verification-status-enum';

export class QrCodeVerificationResponse {
  id!: number;
  uid!: string;
  sl!: number;
  verificationScanStatusEnum!: VerificationStatusEnum;
  qrVerificationCountScan!: number;
  verificationSmsStatusEnum!: VerificationStatusEnum;
  qrVerificationCountSms!: number;
  customerFeedbackSubmitted!: boolean;
}

File: qr-auth-service/src/main/java/com/app/authentiScan/payload/response/QrCodeVerificationResponse.java
package com.app.authentiScan.payload.response;


import com.app.authentiScan.enums.VerificationStatusEnum;
import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class QrCodeVerificationResponse {
    private Long id;
    private String uid;
    private Integer sl;
    private VerificationStatusEnum verificationScanStatusEnum;
    private Integer qrVerificationCountScan;
    private LocalDateTime lastQrVerificationScanAt;
    private VerificationStatusEnum verificationSmsStatusEnum;
    private Integer qrVerificationCountSms;
    private LocalDateTime lastQrVerificationSmsAt;
    private boolean customerFeedbackSubmitted = false;
}
